src/SourceAnnotation.h

```diff
@@ -101,7 +101,8 @@ namespace snowcrash {
         NoError = 0,
         ApplicationError = 1,
         BusinessError = 2,
-        ModelError = 3
+        ModelError = 3,
```

Comment:
In a future I would merge these two into one – they were basically created for "referencing undefined data structure", I guess it is OK for now to leave it as this.